contracts/party/PartyGovernanceNFT.sol

```diff
@@ -356,33 +376,29 @@ contract PartyGovernanceNFT is PartyGovernance, ERC721, IERC2981 {
         {
             IERC20 prevToken;
             for (uint256 i; i < withdrawTokens.length; ++i) {
```

Comment:
While not fully related to this pr, having two separate loops iterating over `withdrawTokens` is bad for readability and gas efficiency.

Reply:
How would you optimize it?